libs/community/tests/integration_tests/test_check_grounding.py

```diff
@@ -0,0 +1,116 @@
+                'to power in Germany. Horrified by the Nazi "war of extermination" '
+                "against his fellow Jews,[12] Einstein decided to remain in the US, "
+                "and was granted American citizenship in 1940.[13] On the eve of World "
+                "War II, he endorsed a letter to President Franklin D. Roosevelt "
+                "alerting him to the potential German nuclear weapons program and "
+                "recommending that the US begin similar research. Einstein supported "
+                "the Allies but generally viewed the idea of nuclear weapons with "
+                "great dismay.[14]"
+            ),
+            metadata={
+                "language": "en",
+                "source": "https://en.wikipedia.org/wiki/Albert_Einstein",
+                "title": "Albert Einstein - Wikipedia",
+            },
+        ),
+        Document(
+            page_content=(
+                "Life and career\n"
+                "Childhood, youth and education\n"
+                "See also: Einstein family\n"
+                "Einstein in 1882, age\xa03\n"
+                "Albert Einstein was born in Ulm,[19] in the Kingdom of Württemberg "
+                "in the German Empire, on 14 March 1879.[20][21] His parents, secular "
+                "Ashkenazi Jews, were Hermann Einstein, a salesman and engineer, and "
+                "Pauline Koch. In 1880, the family moved to Munich's borough of "
+                "Ludwigsvorstadt-Isarvorstadt, where Einstein's father and his uncle "
+                "Jakob founded Elektrotechnische Fabrik J. Einstein & Cie, a company "
+                "that manufactured electrical equipment based on direct current.[19]\n"
+                "Albert attended a Catholic elementary school in Munich from the age "
+                "of five. When he was eight, he was transferred to the Luitpold "
+                "Gymnasium, where he received advanced primary and then secondary "
+                "school education.[22]"
+            ),
+            metadata={
+                "language": "en",
+                "source": "https://en.wikipedia.org/wiki/Albert_Einstein",
+                "title": "Albert Einstein - Wikipedia",
+            },
+        ),
+    ]
+
+
+@pytest.fixture
+def grounded_generation_service_client() -> (
+    discoveryengine_v1alpha.GroundedGenerationServiceClient
+):
+    return discoveryengine_v1alpha.GroundedGenerationServiceClient()
+
+
+@pytest.fixture
+def output_parser(
+    grounded_generation_service_client: (
+        discoveryengine_v1alpha.GroundedGenerationServiceClient
+    ),
+) -> VertexCheckGroundingWrapper:
+    return VertexCheckGroundingWrapper(
+        project_id=os.environ["PROJECT_ID"],
+        location_id=os.environ.get("REGION", "global"),
+        grounding_config=os.environ.get("GROUNDING_CONFIG", "default_grounding_config"),
+        client=grounded_generation_service_client,
+    )
+
+
+def test_integration_parse(
```

Comment:
should any additional APIs be enabled on the project where tests are running?

Reply:
I mean, should any GCP APIs be enabled / whitelisted for the GCP project where this test is running at?

Reply:
Yes, we would need to whitelist Discovery Engine API if it has not been whitelisted for this to run.

Reply:
we need to mark it as `extended` then